event_questions.md

```diff
@@ -0,0 +1,70 @@
+# Selected questions and answers
+
+These are some selected Q&A during the live event we held, anonymized for privacy reasons.
+
+## What is the Github link to the repository?
+
+https://github.com/MLWorkshops/mlinterviews
+
+## Where are the setup instructions?
+
+https://github.com/MLWorkshops/mlinterviews
```

Comment:
Maybe add the section link as in https://github.com/MLWorkshops/mlinterviews#getting-started?